DMCompiler/DMStandard/_Standard.dm

```diff
@@ -8,6 +8,7 @@ proc/arcsin(X)
 proc/arctan(A)
 proc/ascii2text(N)
 proc/ckey(Key)
+proc/ckeyEx(Key)
```

Comment:
According to the [reference page](http://www.byond.com/docs/ref/#/proc/ckeyEx), this parameter's name is `Text`
```suggestion
proc/ckeyEx(Text)
```